.reuse/dep5

```diff
@@ -1,40 +1,31 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: Gardener <repo name>
+Upstream-Name: Gardener oidc-webhook-authenticator
 Upstream-Contact: The Gardener project <gardener@googlegroups.com>
-Source: https://github.com/gardener/<repo name>
+Source: https://github.com/gardener/oidc-webhook-authenticator
 
 # --------------------------------------------------
 # source code
 
-Files:
-  .github/*
-  .gitignore
-  CODEOWNERS
-  go.mod
-  go.sum
-  VERSION
-  PROJECT
-  .gitguardian.yaml
-  .dockerignore
-  .devcontainer/*
-Copyright: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+Files: *
+Copyright: 2017-2024 SAP SE or an SAP affiliate company and Gardener contributors
 License: Apache-2.0
 
-Files:
-  forked/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go
-Copyright: 2015 The Kubernetes Authors
-License: Apache-2.0
 # --------------------------------------------------
 # documentation
 
-Files:
-  CONTRIBUTING.md
-  README.md
-  config/*
-  docs/*
-  cfssl/*
-Copyright: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+Files: *.md
```

Comment:
Tthe docs folder contains files with `.dot` and `svg` extensions which IMO are part of the documentation. WDYT of adding the docs folder explicitly to this rule?

```suggestion
Files:
  docs/*
  *.md
```